inland-empire-software-development/main
# Feature - External links should open in a new tab for navigation

Fixes #103 

Submission information

- [ ] Testing steps have been added to this issue or the PR for this issue
- [ ] Submission has been tested in all supported browsers

New Feature Submissions:
- [ ] Code gone through a code review by at least one other person
- [ ] Code has been locally linted yarn lint


Steps to test:
- Checkout the mission-goal-api-update branch `git checkout feature-target-blank-mobile`
- Run `yarn install && yarn dev` 
- Visit `localhost:3000`
- Visit the main page for the application
- ( If on mobile) Click on the menu 
-  Click on the  nav items - check that **ONLY** EVENTS and JOIN opens a new tab
